packages/coresandbox/src/index.ts

```diff
@@ -171,6 +174,13 @@ const applyConfig = (config: ConfigType) => {
   config.blocks.blocksConfig.listing = listing(config);
   config.views.contentTypesViews.Folder = NewsAndEvents;
 
+  config.registerSlotComponent({
+    slot: 'aboveContent',
+    name: 'hello.aboveFooter',
```

Comment:
This doesn't sit well with me
slot: 'aboveContent',
name: 'hello.aboveFooter'
I'm thinking of aboveFooter as another slot, not a name part of the aboveContent slot

Reply:
Yeah, leads to confusion. I will amend it.

Reply:
pushed